src/commands/ai/agents/call.ts

```diff
@@ -63,28 +27,41 @@ export default class Call extends Command {
       required: false,
       description: 'name or ID of app (required if alias is used)',
     }),
-    json: flags.boolean({char: 'j', description: 'output response as JSON'}),
+    json: flags.boolean({
+      char: 'j',
+      description: 'output response as JSON',
+      exclusive: ['output'],
+    }),
     optfile: flags.string({
       description: 'additional options for model inference, provided as a JSON config file',
       required: false,
+      exclusive: ['opts'],
+      exactlyOne: ['prompt', 'messages', 'opts'],
```

Comment:
I don't think you need both `exclusive` and `exactlyOne` here. `exactlyOne` should cover the situation, saying "you can only use one of these flags".

Reply:
```suggestion
      exclusive: ['opts'],
      exactlyOne: ['prompt', 'messages'],
```